configuration.md

```diff
@@ -21,5 +22,39 @@ The DSpace server will have a whitelist of properties that can be retrieved by t
 }
 ```          
 
-* 200 OK - if the operation succeed
-* 404 Not found - if the property doesn't exist or isn't configured to be retrieved
\ No newline at end of file
+* 200 OK - if the operation succeeded
+* 404 Not found - if the property doesn't exist or isn't configured to be exposed
+
+## Search methods
+**/api/config/properties/search/isExposed**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+
+This endpoint returns all exposed configuration variables.
+
+For example:
+
+```json
+[{
+  "name": "google.analytics.key",
+  "values": [ 
+      "UA-XXXXXX-X"
+   ]
+},
+{
+  "name": "websvc.opensearch.autolink",
+  "values": [ 
+      true
+   ]
+},
+{
+  "name": "orcid.authorize-url",
+  "values": [ 
+      "https://sandbox.orcid.org/oauth/authorize"
+   ]
+}]
+```
+
+* 200 OK - if the operation succeeded
+* 404 Not found - if no property is configured to be exposed
```

Comment:
this should be an empty list but still 200.
The endpoint must support pagination parameters according to our general rules https://github.com/DSpace/RestContract?tab=readme-ov-file#pagination